po/extra/nl.po

```diff
@@ -108,35 +104,35 @@ msgstr "Pictogram verbeterd (veel dank aan @Fatih20)"
 
 #: data/konbucase.appdata.xml.in:90
 msgid "Fix changing OS-wide dark style preference is not reflected"
-msgstr ""
+msgstr "Opgelost: donker systeemthema werd niet overgenomen"
 
 #: data/konbucase.appdata.xml.in:91 data/konbucase.appdata.xml.in:102
 msgid "Update translations"
 msgstr "Vertalingen bijgewerkt"
 
 #: data/konbucase.appdata.xml.in:98
 msgid "Fix the app freezes when trying to input texts in result_combo_entry"
-msgstr ""
+msgstr "Opgelost: vastlopen na tekstinvoer in result_combo_entry"
 
 #: data/konbucase.appdata.xml.in:99
 msgid "Respect an elementary OS-wide dark preference"
-msgstr ""
+msgstr "Ondersteuning voor elementary OS' donkere thema"
 
 #: data/konbucase.appdata.xml.in:100
 msgid "Add a description tooltip to each case"
-msgstr ""
+msgstr "Beschrijvingshulpballonnen toegevoegd"
 
 #: data/konbucase.appdata.xml.in:101
 msgid "Allow settin per-app dark style preference"
-msgstr ""
+msgstr "Ondersteuning voor donker thema per toepassing"
 
 #: data/konbucase.appdata.xml.in:109
 msgid "Add French translation (thanks to @NathanBnm)"
-msgstr "Franse vertaling (met dank aan Nathan Bonnemains)"
+msgstr "Franse vertaling (met dank aan <a class=
```

Comment:
```suggestion
msgstr "Franse vertaling (met dank aan @NathanBnm)"
```

Seems like you pasted wrong text  :sweat_smile:

Reply:
I didn't paste anything - I clicked "Commit suggestion".

Reply:
GitHub says that the original text is good now, so all should be good.

Reply:
Hmm, not sure what happened here but now it is fixed.